src/library/scala/collection/immutable/Map.scala

```diff
@@ -126,8 +126,10 @@ trait MapOps[K, +V, +CC[X, +Y] <: MapOps[X, Y, CC, _], +C <: MapOps[K, V, CC, C]
   */
 object Map extends MapFactory[Map] {
 
+  // getenv not getProperty for Scala.js friendliness.
+  // TODO remove before 2.13.0-RC1? see scala/collection-strawman#572
   private final val useBaseline: Boolean =
-    scala.sys.props.get("scala.collection.immutable.useBaseline").contains("true")
+    System.getenv("SCALA_COLLECTION_IMMUTABLE_USE_BASELINE") == "true"
```

Comment:
I’m not sure we want to keep this switch for 2.13, anyway. Do we? This was introduced mainly for testing.

Reply:
tracking that at https://github.com/scala/collection-strawman/issues/572